modules/ai/assets/js/editor/pages/form-layout/components/prompt-autocomplete.js

```diff
@@ -24,11 +25,56 @@ TextInput.propTypes = {
 	InputProps: PropTypes.object,
 };
 
+const PaperComponent = function( props ) {
+	return <Paper { ...props }
+		sx={ {
+			borderRadius: 2,
+		} }
+	>
+		<Typography
+			component={ Box }
+			color={ ( theme ) => theme.palette.text.tertiary }
+			variant="caption"
+			paddingX={ 2 }
+			paddingY={ 1 }
+		>
+			{ __( 'Suggested Prompts', 'elementor' ) }
+		</Typography>
+		<Divider />
+		{ props.children }
+	</Paper>;
+};
+
+PaperComponent.propTypes = {
+	children: PropTypes.node,
+};
+
 const PromptAutocomplete = ( { onSubmit, ...props } ) => {
 	const [ showSuggestions, setShowSuggestions ] = useState( false );
+	const theme = useTheme();
+	const itemHeight = parseInt( theme.spacing( 4 ) );
+	const maxItems = 5;
 
 	return (
 		<Autocomplete
+			PaperComponent={ PaperComponent }
+			ListboxProps={ { sx: { maxHeight: maxItems * itemHeight } } }
```

Comment:
```suggestion
			ListboxProps={ { sx: { maxHeight: ( theme ) => maxItems * theme.spacing( 4 ) } } }
```

maybe?
instead of having the `useTheme` hook

see:
https://mui.com/system/getting-started/the-sx-prop/#callback-values

Reply:
It's for self documenting code.